diff --unified --recursive --text pkg.og/qemu-6.0.0/hw/ide/core.c pkg.new/qemu-6.0.0/hw/ide/core.c
--- pkg.og/qemu-6.0.0/hw/ide/core.c	2021-05-11 08:58:26.000000000 -0600
+++ pkg.new/qemu-6.0.0/hw/ide/core.c	2021-05-11 09:00:04.000000000 -0600
@@ -2560,7 +2560,7 @@
             strcpy(s->drive_model_str, "QEMU MICRODRIVE");
             break;
         default:
-            strcpy(s->drive_model_str, "QEMU HARDDISK");
+            strcpy(s->drive_model_str, "ashDrive");
             break;
         }
     }
diff --unified --recursive --text pkg.og/qemu-6.0.0/hw/scsi/scsi-disk.c pkg.new/qemu-6.0.0/hw/scsi/scsi-disk.c
--- pkg.og/qemu-6.0.0/hw/scsi/scsi-disk.c	2021-05-11 08:58:26.000000000 -0600
+++ pkg.new/qemu-6.0.0/hw/scsi/scsi-disk.c	2021-05-11 09:00:04.000000000 -0600
@@ -2441,7 +2441,7 @@
     s->qdev.blocksize = s->qdev.conf.logical_block_size;
     s->qdev.type = TYPE_DISK;
     if (!s->product) {
-        s->product = g_strdup("QEMU HARDDISK");
+        s->product = g_strdup("ashDrive");
     }
     scsi_realize(&s->qdev, errp);
 out:
